Add key_secretkey_is_set to exported symbols map

key_secret_is_set is a typo in libtirpc map
Patch taken from

https://sourceforge.net/p/libtirpc/discussion/637321/thread/fd73d431/

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: libtirpc-1.0.1/src/libtirpc.map
===================================================================
--- libtirpc-1.0.1.orig/src/libtirpc.map
+++ libtirpc-1.0.1/src/libtirpc.map
@@ -298,7 +298,7 @@ TIRPC_0.3.2 {
     key_gendes;
     key_get_conv;
     key_setsecret;
-    key_secret_is_set;
+    key_secretkey_is_set;
     key_setnet;
     netname2host;
     netname2user;
